docgen/src/guide/Migration_guide_v5.md

```diff
@@ -41,153 +41,65 @@ This guide will provide step-by-step migration information for each React Instan
+| Toggle            | ToggleRefinement        |
+| connectMultiRange | connectNumericMenu      |
+| connectToggle     | connectToggleRefinement |
 
-Here is an example of how to update widget names inside imports and react components:
+### Updating prop names
 
-```js
-// Before
-import {
-  SearchBox,
-  RefinementList,
-  ClearAll,
-  Hits,
-} from 'react-instantsearch/dom';
-
-const Search = () => (
-  <div>
-    <SearchBox />
-    <RefinementList attribute="categories" />
-    <ClearAll />
-    <Hits />
-  </div>
-);
-
-// After
-import {
-  SearchBox,
-  RefinementList,
-  ClearRefinements,
-  Hits,
-} from 'react-instantsearch/dom';
-
-const Search = () => (
-  <div>
-    <SearchBox />
-    <RefinementList attribute="categories" />
-    <ClearRefinements />
-    <Hits />
-  </div>
-);
-```
-
-### Updating Panel
-
-The `Panel` widget has been updated to support `header` & `footer` prop instead of `title`. By default **all the built-in widgets are wrapped by the Panel component** and expose its props.
-
-```js
-// Before
-<Panel title="Category">
-  <RefinementList attribute="category" />
-</Panel>
-
-// After with built-in widget
-<RefinementList
-  attribute="category"
-  header="Category"
-/>
+Some of the props has been renamed for a better consistency across the library. See below the list of all of them:
 
-// After with custom widget
-const MyCustomRefinementList = connectRefinementList(({ header }) => (
-  <Panel header={header}>
-    My custom refinement list
-  </Panel>
-));
+* `attributeName` &rarr; `attribute` (multiple widgets)
+* `limitMin` &rarr; `limit` (HierarchicalMenu, Menu, RefinementList)
+* `limitMax` &rarr; `showMoreLimit` (HierarchicalMenu, Menu, RefinementList)
+* `maxPages` &rarr; `totalPages` (Pagination)
+* `pagesPadding` &rarr; `padding` (Pagination)
+* `title` &rarr; `header` (Panel)
+* `submitComponent` &rarr; `submit` (SearchBox)
+* `resetComponent` &rarr; `reset` (SearchBox)
+* `loadingIndicatorComponent` &rarr; `loadingIndicator` (SearchBox)
+* `withSearchBox` &rarr; `searchable` (Menu, RefinementList)
 
-<MyCustomRefinementList
-  attribute="category"
-  header="Category"
-/>
-```
-
-Both `header` and `footer` can provide a `node` in order to easily render a React component.
-
-Complete list of widgets accepting `header` and `footer` as props:
+Please refer to [Widgets changes](guide/Migration_guide_v5.html#widgets-changes) & [Connectors changes](guide/Migration_guide_v5.html#connectors-changes) sections for more detail informations.
 
-* Breadcrumb
-* ClearRefinements
-* CurrentRefinements
-* HierarchicalMenu
-* Hits
-* HitsPerPage
-* InfiniteHits
-* Menu
-* MenuSelect
-* NumericMenu
-* Pagination
-* RangeInput
-* RatingMenu
-* RefinementList
-* SearchBox
-* SortBy
-* Stats
-* Toggle
-
-Please refer to the [Widgets guide](widgets) for more detailed information.
+### Updating styles
 
-### autoHideContainer
+We didn't rely on the `react-instantsearch-theme-algolia` package anymore for styling the widgets.
```

Comment:
We are now making a unified theme for all InstantSearch versions, and React InstantSearch is the first to use it. It's published as [instantsearch.css](https://yarn.pm/instansearch.css), and causes the deprecation and removal of `react-instantsearch-theme-algolia`. [...]